lib/mongodb.js

```diff
@@ -60,8 +60,24 @@ exports.initialize = function initializeDataSource(dataSource, callback) {
 
   var s = dataSource.settings;
 
-  s.safe = (s.safe !== false);
-  s.w = s.w || 1;
+  // create db property if it doesn't exist yet
+  s.db = s.db || {};
+
+  // define safe writeConcern default values
+  // https://docs.mongodb.org/v3.0/reference/write-concern/
+  var safeOptions = {
+    w: 1,               // write to standalone server or at least primary in replSet
+    j: true             // request acknowledgement from MongoDB that the write operation has been written to the journal
+    // wtimeout: 1000   // timeout is only applicable for w > 1
+  };
+
+  // when safe option is set to "true", assign default values for "w" and "j" to db options
+  if (s.safe === true) {
```

Comment:
the documented datasource settings would allow to define `dataSource.settings.safe`, which is exactly the reason for this PR. when setting `dataSource.settings.safe` to true, none of the safe options would have been forwarded / interpreted by `mongodb`. So, this PR allows to continue setting `dataSource.settings.safe` and actually get the settings activated that were intended. I assume, the original intention for this setting here was to have sth similar to what [mongoose allows with their `safe` option on the connection object](http://mongoosejs.com/docs/guide.html#safe).

so no, it shouldn't be `if (s.db.safe === true)`